flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowFormatFactoryTest.java

```diff
@@ -128,6 +152,14 @@ private void testSchemaDeserializationSchema(Map<String, String> properties) {
 		assertEquals(expected2, actual2);
 	}
 
+	private void testSchemaDeserializationSchemaIgnoreParseErrors(Map<String, String> properties) {
```

Comment:
We don't need to extract a method if this method is only be invoked in one place.

Reply:
ok , i understand.

Reply:
i have resolved this.